src/infrastructure/api/index.ts

```diff
@@ -151,6 +163,50 @@ router.get(
   },
 );
 
+router.get('/tfl', (_req, res) => {
+  let secretsManager: SecretsManagerServiceInterface;
+  if (process.env.IS_OFFLINE === 'true') {
+    logger.debug('configuring local secret manager');
+    secretsManager = new LocalSecretsManagerService();
+  } else {
+    logger.debug('configuring aws secret manager');
+    secretsManager = new SecretsManagerService(new AWS.SecretsManager());
```

Comment:
Are we getting secrets on every call? If so, do we need to?
It would be good to have it done once per lambda startup, but if the feed only runs once a day there's little benefit.

Reply:
I take your point it would make more sense to do so, but yeah as you said it only runs once a day so maybe not worth the investment. It also would mean again changing the other end points to meet this new style and we agreed as a team to not look at refactoring this work too much due to time limitations. So in summary, I think I will leave this as is.